### PR TITLE
Add log4cpp recipe

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Add log4cpp recipe

http://log4cpp.sourceforge.net

Log4cpp is library of C++ classes for flexible logging to files, syslog, IDSA and other destinations. It is modeled after the Log4j Java library, staying as close to their API as is reasonable.

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
@jakirkham , How do I have to proceed in this case when already exist a package in ```bioconda``` ?

**Comment by nsoranzo** [comment]
@marcelotrevisani For packages that are not specific to biological sciences (like this one), conda-forge is usually the preferred place. Once this is merged, please open a pull request to remove it from the bioconda-recipes repository.

**Comment by marcelotrevisani** [comment]
I didn't know! Thanks for this info @nsoranzo !

**Comment by marcelotrevisani** [comment]
@conda-forge-admin , please rerender

**Comment by marcelotrevisani** [comment]
@conda-forge/staged-recipes could someone review and merge that for me?

File: recipes/log4cpp/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./configure --prefix="${PREFIX}"
+make -j${CPU_COUNT}
+make install
```
Comment (scopatz) [inline recipes/log4cpp/build.sh]: Is there a `make check` you can run?

**Comment by scopatz** [comment]
Looks good @marcelotrevisani, the make check thing is just a suggestion, since the tests are more thorough than existence tests.

File: recipes/log4cpp/build.sh (hunk 1 above)
Comment (marcelotrevisani) [reply recipes/log4cpp/build.sh]: There is a ``make check``, I was using that but I removed. However I will add that again.

Thanks! :)

**Pull request merged**